packages/babylon/src/util/charCodes.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+type Char = number;
+
+const charCodes = {
```

Comment:
instead of exporting an object it would be better to export them as named exports to support better tree-shakeability of those

Reply:
Good point.

Reply:
That would also enable better inlining of this values i think.